srcpkgs/PrismLauncher/template

```diff
@@ -0,0 +1,42 @@
+# Template file for 'PrismLauncher'
+pkgname=PrismLauncher
+version=5.0
+revision=1
+build_style=cmake
+configure_args="-DLauncher_BUILD_PLATFORM=Void"
+hostmakedepends="extra-cmake-modules openjdk8 pkg-config qt5-host-tools
+ qt5-qmake scdoc"
+makedepends="qt5-devel"
+depends="virtual?java-runtime qt5-svg qt5-imageformats xrandr"
```

Comment:
@oynqr why does this depend on xrandr?

Reply:
ok the only reference I can find to it no longer exists in the repo: https://github.com/PrismLauncher/PrismLauncher/commit/663a1a5a8383dd60ef3cf2c54ba8e8edca12c552

Reply:
hm i see. since it's not needed for every version, maybe it could be mentioned in the README.voidlinux instead? I'll defer to you as package maintainer

Reply:
I don't remember how descriptive the error messages for missing xrandr were and don't really have time to check right now, but a message in the README.voidlinux should suffice.